src/widget/wtrackmenu.cpp

```diff
@@ -1737,7 +1750,8 @@ void WTrackMenu::slotPurge() {
 }
 
 void WTrackMenu::clearTrackSelection() {
-    m_trackPointerList.clear();
+    m_pTrack = nullptr;
+    m_deckGroup.clear();
```

Comment:
`= QString()` for consistency? Initially `m_deckGroup` is a null-string, not an empty string.

Reply:
ok